did/plugins/github.py

```diff
@@ -9,6 +9,7 @@
     url = https://api.github.com/
     token = <authentication-token>
     login = <username>
+    project = <project1, project2>
```

Comment:
```suggestion
    project = <project1>, <project2>
```

Let's make it a bit more clear that comma is a project separator here.